pkg/server/events.go

```diff
@@ -118,13 +118,14 @@ func convertEvent(e *gogotypes.Any) (string, interface{}, error) {
 // start starts the event monitor which monitors and handles all container events. It returns
 // a channel for the caller to wait for the event monitor to stop. start must be called after
 // subscribe.
-func (em *eventMonitor) start() (<-chan struct{}, error) {
+func (em *eventMonitor) start() <-chan error {
+	errCh := make(chan error)
 	if em.ch == nil || em.errCh == nil {
-		return nil, errors.New("event channel is nil")
+		logrus.Fatal("event channel is nil")
```

Comment:
If this case is a programmer error, not initializing channels, then you should `panic()` not log fatal

Reply:
Will do. Thought panic and fatal are same thing.

Reply:
Same outcome yes, but panic means "I messed up in the code" and a log message with an exit(1) usually just means an application/runtime error, not developer error

Reply:
Thanks, :D Got it.

Reply:
Done.